### PR TITLE
fix: pfelement - restore pfelement tests

### PR DESCRIPTION
**Pull request by mwcz**
## pfelement

The pfelement tests weren't running, this restores them so they'll be run along with the rest of the test suite.

### What has changed and why

pfelement tests weren't running, seemingly because they were attempting to import pfelement.js from the package root, instead of the dist directory.

### Testing instructions

Try running `npm test` and look for pfelement tests, or `npm test pfelement` and verify that more than 0 tests run (previously, it would "pass" with zero tests run).

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Tests have been updated to cover these changes.
- [ ] Browser testing passed.
- [x] Repository compiles and tests pass.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfelement/src/pfelement.scss [hunk 1]
```diff
@@ -12,15 +12,20 @@ body {
 
 body[unresolved] {
   opacity: 0;
+  visibility: hidden;
```
Comment (kylebuch8) [inline elements/pfelement/src/pfelement.scss:15]: @mwcz - I don't think you meant to include the CSS changes in this PR. Can you pull these out?

Comment (mwcz) [reply elements/pfelement/src/pfelement.scss:15]: Whoops, thanks, I'll pull those out.

Comment (kylebuch8) [reply elements/pfelement/src/pfelement.scss]: After merging master, this doesn't matter any more.
